src/App.js

```diff
@@ -1,13 +1,91 @@
-import React from 'react';
+import React from "react";
+import ToDoList from "./components/TodoComponents/TodoList";
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      toDoArray: [
```

Comment:
Try not to include the data type in the variable name, e.g `toDoArray` would better be `todos`